doordeck-sdk/src/androidMain/kotlin/com/doordeck/multiplatform/sdk/api/AccountApi.android.kt

```diff
@@ -10,142 +10,170 @@ import com.doordeck.multiplatform.sdk.model.responses.UserDetailsResponse
 import com.doordeck.multiplatform.sdk.util.completableFuture
 import java.util.concurrent.CompletableFuture
 
+/**
+ * Platform-specific implementations of account-related API calls.
+ */
 actual object AccountApi {
     /**
-     * Refresh token
-     *
-     * @see <a href="https://developer.doordeck.com/docs/#refresh-token">API Doc</a>
+     * @see AccountClient.refreshTokenRequest
```

Comment:
I don't think it's worth changing everything because it would be rather fiddly, but I think the 'correct' JavaDoc approach would be:
```suggestion
     * @see AccountClient#refreshTokenRequest(String)
```
It would be clickable in IntelliJ if recognized as JavaDoc, or are you aiming for another doc scheme? Are we generating JavaDoc for JVM platforms?